runtime/queries/markdown/highlights.scm

```diff
@@ -10,15 +10,16 @@
   (fenced_code_block)
 ] @markup.raw.block
 
+(block_quote) @markup.quote
+
 (code_span) @markup.raw.inline
 
 (emphasis) @markup.italic
 
 (strong_emphasis) @markup.bold
 
-(link_destination) @markup.underline.link
```

Comment:
This was per https://www.sublimetext.com/docs/scope_naming.html#markup but I guess we could diverge here

Reply:
The name seemed weird to me and `markup.underline.label` didn't make much sense :P Since we don't guarantee compatibility with textmate scopes I think it's okay to diverge.